### PR TITLE
[CLD-6487] Replace 'Country' with 'Country/Region' in country selectors

### PR DESCRIPTION
**Pull request by nickmisasi**
#### Summary
Changes occurrences of "Country" to "Country/Region" in the country selectors. Overrides ISO3166 country name for a certain island in the pacific.

This change applies to anywhere the country selector is used in product - including
Cloud Purchase Modal
Self hosted purchase modal
Payment method page in the system console
Start trial form modal
etc

#### Ticket Link
https://mattermost.atlassian.net/browse/CLD-6487

#### Screenshots
image
image

#### Release Note

```release-note
None
```

### CODE REVIEW

File: webapp/channels/src/actions/cloud.tsx [hunk 1]
```diff
@@ -48,7 +48,7 @@ export function completeStripeAddPaymentMethod(
                             line2: billingDetails.address2,
                             city: billingDetails.city,
                             state: billingDetails.state,
-                            country: getCode(billingDetails.country),
+                            country: billingDetails.country,
```
Comment (nickmisasi) [inline webapp/channels/src/actions/cloud.tsx:51]: The country field is already storing it as the ISO code, so there was no need for this getCode call.

File: webapp/channels/src/utils/countries.ts [hunk 2]
```diff
@@ -1,11 +1,16 @@
...
 
 type Country = {
     name: string;
     code: string;
 }
 
+overwrite([{
+    code: 'TW',
+    name: 'Taiwan',
+}]);
```
Comment (M-ZubairAhmed) [inline webapp/channels/src/utils/countries.ts:14]: why is this?

**Comment by johndavidlugtu** [comment]
@nickmisasi can you list in the description areas in the product and customer portal where the changes apply for easier visibility?

**Review by M-ZubairAhmed — APPROVED** [review]

**Comment by nickmisasi** [comment]
@johndavidlugtu updated

**Review by johndavidlugtu — APPROVED** [review]
Thanks Nick

**Review by yasserfaraazkhan — APPROVED** [review]

**Pull request merged**